spiel/ut/spiel_ci_ut.c

```diff
@@ -676,7 +684,22 @@ static void test_spiel_pool_repair(enum m0_repreb_type type)
 	M0_UT_ASSERT(status[0].srs_progress > 0);
 	m0_free0(&status);
 
-	spiel_repair_start(&pool_fid, &svc_fid, type);
+	spiel_ci_ut_ha_state_set(&io_disk, M0_NC_FAILED);
```

Comment:
Please let me know the need of these changes?

Reply:
This is required for the following sns_spiel_repair_quiesce() to succeed.